pom.xml

```diff
@@ -9,13 +9,12 @@
     <artifactId>kafka-sink-azure-kusto</artifactId>
     <packaging>jar</packaging>
     <description>A Kafka Connect plugin for Azure Data Explorer (Kusto) Database</description>
-    <version>3.0.8</version>
+    <version>4.0.0</version>
```

Comment:
I think this need a major release (?)

Reply:
Yes right, this will need to be a new release.